kubernetes-ingress.md

```diff
@@ -0,0 +1,150 @@
+# Ingress
+[⬅️ Back to Kubernetes overview](README.md)
+
+
+## Setup ingress controller
+![Ingress Controller](./kubernetes/ingress.png)
+
+### When using kind:
+From Kind [Documentation](https://kind.sigs.k8s.io/docs/user/ingress/)
+1. Setup Cluster
+
+```sh
+kind delete cluster #if one is alrady created
+cat <<EOF | kind create cluster --config=-
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  kubeadmConfigPatches:
+  - |
+    kind: InitConfiguration
+    nodeRegistration:
+      kubeletExtraArgs:
+        node-labels: "ingress-ready=true"
+  extraPortMappings:
+  - containerPort: 80
+    hostPort: 80
+    protocol: TCP
+  - containerPort: 443
+    hostPort: 443
+    protocol: TCP
+EOF
+```
+> [!WARNING]
+> you may need to change the hostport when some other service is already using it
+
+2. Create Namespace
+
+```sh
+kubectl create namespace ingress-nginx
```

Comment:
The resource you link below already creates a namespace, but I think showing that it gets its own namespace is good